q2_types/feature_data/_format.py

```diff
@@ -144,69 +143,15 @@ def _validate_(self, level):
 
 
 class DNAFASTAFormat(model.TextFileFormat):
-    def _validate_lines(self, max_lines):
+    def _validate_(self, max_lines):
+        level_map = {'min': 100, 'max': float('inf')}
```

Comment:
How about you move this into `_validate_DNAFASTAFormats`, and pass the `level` in as an argument?